Lib/tarfile.py

```diff
@@ -2461,9 +2461,14 @@ def __exit__(self, type, value, traceback):
 def is_tarfile(name):
     """Return True if name points to a tar archive that we
        are able to handle, else return False.
+
+       The name argument may be a file or file-like object.
```

Comment:
Change last sentence to:
<pre>
'name' should be a string, file, or file-like object.
</pre>